wiki/Tournaments/GTS/BGTS_2020/en.md

```diff
@@ -66,7 +66,7 @@ The Beginners Global Taiko Showdown 2020 is run by various community members.
 ## Participants
 
 |  | Country | Members |
-| --: | :-- | :-- |
+| :-: | :-- | :-- |
```

Comment:
this was intentional by me last time, think it looks nicer to align the flags rather than varying alignment with center (?)

either way, why is it in this pr 🤔

Reply:
Hate opening new PR to just change 1 piece of char.
If it's going to be right aligned in the future, so be it.